README.md

```diff
@@ -144,15 +151,15 @@ export default function () {
 ```
 
 ```javascript
-import { upload } from '../../helpers/upload';
+/* global upload */
```

Comment:
I'm a bit behind on my own projects' `ember-cli` version, so I don't know how to go about either `import` this helper (which we're now declaring with `Ember.Test.registerAsyncHelper`) or configuring our linter to consider it globally accessible.

Addons used to modify the `"global"` key in projects' `.jshintrc` and `tests/.jshintrc` files, but now we use `.eslintrc`.

What do you suggest?

Reply:
I definitely need to go and update this to Ember 2.18, since it's the last LTS before 3.0

Reply:
I think this is alright how you did it